app/services/menu/destroy.rb

```diff
@@ -0,0 +1,19 @@
+class Menu
+  class Destroy < ApplicationService
+    def initialize(params = {})
```

Comment:
Why do you need default param here?
What if there's no `id` in `params` hash?
Calling `menu.destroy` will result in an error